corehq/apps/domain/models.py

```diff
@@ -363,6 +363,11 @@ def wrap(cls, data):
             self.save()
         return self
 
+    def get_default_timezone(self):
+        """return a timezone object from self.default_timezone"""
+        import pytz
```

Comment:
not really than that I expect this to be somewhat short-term and it's only called in one place. I can move it to the top. One sec